### DISCUSSION TITLE
In confluence document how to we @mention someone?

### DISCUSSION THREAD

**Discussion by fkgruber** [quarto-dev/quarto-cli#10949] [Q&A] [answered]
### Description

 I was trying somethying like
````
```{=confluence}
<ac:link>
<ri:user ri:username="name.surname"/>
</ac:link>
```
````
but I get @Unlicensed user

**Comment by mcanouil** [comment]
I believe you asked your question in the wrong place.
This repository is about Quarto CLI, not about Confluence syntax/API.

I suggest to go look in places dedicated to Confluence.

**Reply by cscheid** [reply]
I think it's more likely that they'll find answers in those pages, but this _is_ a question Quarto users are likely to run into. I actually wonder if we need a shortcode or something to help in this situation.

**Reply by mcanouil** [reply]
We could add links to Confluence documentation as well in Quarto documentation.

The "mention code" could be added as example in https://quarto.org/docs/publishing/confluence.html#raw-confluence-blocks

**Reply by fkgruber** [reply]
thank you I also think this is very relevant for the quarto CLI confluence export. This is a basic functionality that any quarto/confluence user is going to need in practice since most work is collaborative.

Ideally we would have some sort of shortcode but if at least we had some example with raw-confluence-blocks that would be helpful.

**Reply by mcanouil** [reply]
There is one example, see the link above.

Note that the example also includes a link to Confluence format syntax (I initially missed it): https://confluence.atlassian.com/doc/confluence-storage-format-790796544.html

From this it seems your code is not valid.

And from a quick search on Confluence: https://confluence.atlassian.com/plugins/servlet/mobile?contentId=771892443#content/view/771892443

From this, it seems that you need the "userkey" and if that is true,  then this makes it way more complicated to make a shortcode out of it.

**Comment by fkgruber** [comment, marked as answer]
I got it working based on the discussion here:
https://community.atlassian.com/t5/Confluence-questions/What-is-the-confluence-source-code-for-a-user-mention/qaq-p/2337124

````
```{=confluence}
<ac:link><ri:user ri:account-id="myid" /></ac:link>
```
````
and myid can be found by creating a confluence page, adding a mention to the user of interest, then use view-source to search for accountId.
